brianlong/www.validators.app
# 186818796 apply skipped after score

#### What's this PR do?
- add skipped after score to views and api
- add skipped after charts colouring based on scores
-  increase max score

#### How should this be manually tested?
- run tests
- check if the api includes skipped_after_score
- check scores views and modal if they all include skipped_after
- check api docs and faq
- check skipped after charts

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/186818796)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
